web/rollup.config.js

```diff
@@ -0,0 +1,93 @@
+import { swc } from "rollup-plugin-swc3";
+import { dts } from "rollup-plugin-dts";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import { terser } from "rollup-plugin-terser";
+import nodePolyfills from 'rollup-plugin-polyfill-node';
+import bakedEnv from 'rollup-plugin-baked-env';
+import pkg from './package.json';
+
+const name = 'lottie-player';
+const globals = {
+  url: "url",
+  lit: "lit",
+  "lit/decorators.js": "lit/decorators.js",
+};
+
+export default [
+  {
+    input: "./src/lottie-player.ts",
+    treeshake: false,
+    output: [
+      {
+        file: './dist/lottie-player.js',
+        format: "umd",
+        name,
+        minifyInternalExports: true,
```

Comment:
you should not minimize the final bundle, the end-consumers can do it themselves. Currently, it's inconvenient, they have to go to the repo to check what exists in the source code while they could do it locally via node_modules

```suggestion
        minifyInternalExports: false,
```

The same applies to other minify configs below

Reply:
This is necessary for the user who doesn't use npm and import via CDN, it reduces library loading time, UMD is only for CDN user, we should minify them.